source/includes/api/_vocabulary.md

```diff
@@ -1,694 +1,1297 @@
+
+## Getting vocabularies and tags across resources
+
+There are several endpoints you can use to retrieve vocabularies and their details, independently from the resources to
+which they are associated
+
+### Getting all vocabularies
+
+> Getting all vocabularies, their resources and tags
 
 ```shell
-curl -X POST https://api.resourcewatch.org/v1/dataset/<dataset-id>/widget/vocabulary/find-by-ids \
--H "Content-Type: application/json"  -d \
- '{
-   "ids": [<ids>]
-  }'
+curl -X GET https://api.resourcewatch.org/v1/vocabulary
 ```
 
-```shell
-curl -X POST https://api.resourcewatch.org/v1/dataset/<dataset-id>/layer/vocabulary/find-by-ids \
--H "Content-Type: application/json"  -d \
- '{
-   "ids": [<ids>]
-  }'
+> Example response
+
+```json
+{
+    "data": [
+        {
+            "id": "test1",
+            "type": "vocabulary",
+            "attributes": {
+                "resources": [
+                    {
+                        "tags": [
+                            "tag1",
+                            "tag2"
+                        ],
+                        "id": "7fa6ec77-5ab0-43f4-9a4c-a3d19bed1e90",
+                        "dataset": "7fa6ec77-5ab0-43f4-9a4c-a3d19bed1e90",
+                        "type": "dataset"
+                    },
+                    {
+                        "tags": [
+                            "tag1",
+                            "tag2"
+                        ],
+                        "id": "0002ed77-f07d-4231-b4eb-cfda77eeafe5",
+                        "dataset": "7fa6ec77-5ab0-43f4-9a4c-a3d19bed1e90",
+                        "type": "layer"
+                    },
+                    {
+                        "tags": [
+                            "tag1",
+                            "tag2"
+                        ],
+                        "id": "0002ed77-f07d-4231-b4eb-cfda77eeafe5",
+                        "dataset": "7fa6ec77-5ab0-43f4-9a4c-a3d19bed1e91",
+                        "type": "layer"
+                    }
+                ],
+                "name": "test1",
+                "application": "rw"
+            }
+        },
+        {
+            "id": "forestChange",
+            "type": "vocabulary",
+            "attributes": {
+                "resources": [],
+                "name": "forestChange",
+                "application": "gfw"
+            }
+        }
+    ]
+}
 ```
 
-> Example of request with real data
+This endpoint will give you a list of vocabularies. Inside each, you'll find a list of resources, identifying the
+resource id, type, associated dataset (if the resource is a dataset, the dataset id will be equal to the resource id)and
+the tags that said vocabulary associates to that resource
+
+You can optionally pass a `limit` integer value as a query parameter if you wish to limit the number of vocabularies
```

Comment:
Is this paginated? If not let's call that out explicitly

Reply:
no, it just supports a simple "limit" parameter.